Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA261 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()

```

Comment:
why you use void main()? in c++ is main() return void datatype?

Reply:
its my mistake. i have to write int main( ) instead of void main() because in cpp main () return integer value.

Reply:
but you have to write return o before getch